python/graphstorm/model/node_glem.py

```diff
@@ -64,6 +105,34 @@ def __init__(self,
         # True: lm is being trained
         # False: gnn is being trained
         self.training_lm = None
+        if lm_param_group is None:
+            lm_param_group = ["pure_lm", "node_proj_matrix"]
+        if gnn_param_group is None:
+            gnn_param_group = ["node_input_projs"]
+        assert set(lm_param_group).issubset(GLEM_CONFIGURABLE_PARAMETER_NAMES['lm_param_group'])
+        assert set(gnn_param_group).issubset(GLEM_CONFIGURABLE_PARAMETER_NAMES['gnn_param_group'])
+        self.param_names_groups = {'lm': lm_param_group, 'gnn': gnn_param_group}
+        # set up default flag for `training_sparse_embed` based on whether its trainable at
+        # either stages:
+        self.training_sparse_embed = 'sparse_embed' in lm_param_group + gnn_param_group
```

Comment:
after this point, is it determined what are included in the LM parameter group and what are included in the GNN parameter group?

Reply:
if it is, i think we should construct these two groups in the init function, instead of building them in the `toggle_params` method.

Reply:
That is correct.